src/reimer_arenkiel_lab_to_nwb/embargo2024/embargo2024_convert_session.py

```diff
@@ -2,86 +2,105 @@
 
 from pathlib import Path
 from typing import Union
-from zoneinfo import ZoneInfo
-from natsort import natsorted
-
+from time import time
+from tqdm import tqdm
 from neuroconv.utils import load_dict_from_file, dict_deep_update
+from neuroconv.tools.nwb_helpers import configure_and_write_nwbfile
 
 from reimer_arenkiel_lab_to_nwb.embargo2024 import Embargo2024NWBConverter
-
+from reimer_arenkiel_lab_to_nwb.dj_utils import *
 
 def session_to_nwb(
-    data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], session_date:str, session_id: str, subject_id:str, stub_test: bool = False
+        data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], key: dict, stub_test: bool = False, verbose: bool = True
 ):
-
     data_dir_path = Path(data_dir_path)
-    folder_path = data_dir_path / Path(session_date)
+    folder_path = data_dir_path / f"{key['animal_id']}_{key['session']}_{key['scan_idx']}"
 
     output_dir_path = Path(output_dir_path)
     if stub_test:
         output_dir_path = output_dir_path / "nwb_stub"
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
-    nwbfile_path = output_dir_path / f"{subject_id}_{session_id}.nwb"
+    nwbfile_path = output_dir_path / f"seb-{key['animal_id']}_ses-{key['session']}.nwb"
```

Comment:
Probably a typo for subject:
```suggestion
    nwbfile_path = output_dir_path / f"sub-{key['animal_id']}_ses-{key['session']}.nwb"
```